app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -1376,6 +1376,24 @@ def _validate_sticky_enrollment(self, data):
 
         return data
 
+    def _validate_rollout_version_support(self, data):
+        if not self.instance or not self.instance.is_rollout:
+            return data
+
+        min_version = NimbusExperiment.Version.parse(self.instance.firefox_min_version)
+        rollout_version_supported = NimbusExperiment.ROLLOUT_SUPPORT_VERSION.get(
+            self.instance.application
+        )
+        if (
+            rollout_version_supported is not None
```

Comment:
Forcing you to check for `None` with `Optional` return values is the only useful thing I've seen mypy do, but it doens't work well with Django so we can't use it 😖

Reply:
And yeah so this will pass quietly if we didn't specify a minimum version for some given application, which may be what we want, may not.